mongodbatlas/resource_mongodbatlas_api_key.go

```diff
@@ -0,0 +1,217 @@
+package mongodbatlas
```

Comment:
can you confirm that we don't need to make any updates to [go-client ](https://github.com/mongodb/go-client-mongodb-atlas)as part of this update?

Reply:
api_keys supported in go-client